cmd/geth/config.go

```diff
@@ -164,7 +164,7 @@ func makeFullNode(ctx *cli.Context) (*node.Node, ethapi.Backend) {
 	}
 	backend, eth := utils.RegisterEthService(stack, &cfg.Eth)
 	// Warn users to migrate if they have a legacy freezer format.
-	if eth != nil {
+	if eth != nil && !ctx.GlobalIsSet(utils.DeveloperFlag.Name) {
```

Comment:
In Line 177, we forcibly stop the Geth if any error is occurred. Can we degrade the log level to `Error`? It's just an additional check and it's OK to fail.

Just in case we still have a few mistakes.